test/plots/piePlotTests.ts

```diff
@@ -412,14 +348,43 @@ describe("Plots", () => {
         svg.remove();
       });
 
-      it("skips invalid Datsets", () => {
+      it("skips invalid Datasets", () => {
         let allSectors = piePlot.selections([new Plottable.Dataset([])]);
         assert.strictEqual(allSectors.size(), 0, "no sectors retrieved");
 
         svg.remove();
       });
 
-      it("retrieves entities under a point with entitiesAt()", () => {
+    });
+
+    describe("entities", () => {
+      let svg: d3.Selection<void>;
+      let dataset: Plottable.Dataset;
+      let simpleData = [{value: 5}, {value: 15}];
+      let piePlot: Plottable.Plots.Pie;
+
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(500, 500);
+        dataset = new Plottable.Dataset();
+        piePlot = new Plottable.Plots.Pie();
+        dataset.data(simpleData);
+        piePlot.addDataset(dataset);
+        piePlot.sectorValue((d) => d.value);
+        piePlot.renderTo(svg);
+      });
+
+      it("returns both a fill path and a stroke path in each Entity's selection", () => {
+        let entities = piePlot.entities();
+        assert.lengthOf(entities, simpleData.length, "returned one Entity per datum");
+        entities.forEach((entity) => {
+          assert.strictEqual(entity.selection.size(), 2, "each entity selection has 2 paths");
+          assert.strictEqual(entity.selection.filter(".fill").size(), 1, "each entity selection has 1 fill path");
+          assert.strictEqual(entity.selection.filter(".outline").size(), 1, "each entity selection has 1 stroke path");
+        });
+        svg.remove();
+      });
+
+      it("retrieves the entity under each given point with entitiesAt()", () => {
         let data = [
```

Comment:
Why are we using a different set of data here? Let's homogenize this and not break the `describe` so badly. My suggestion is to use this set of data instead of the `simpleData` as it does not really matter for the `it` above.

Alternatively, let each test manage their own data and do not specify it at the top level.